### ISSUE TITLE
Incorrectly reporting unused imports for the 'as' syntax

### ISSUE THREAD

**Issue by hvdklauw** [AtomLinter/linter-flake8#253]
**Versions** 

- Flake8: `3.0.4 (flake8-docstrings: 1.0.2, pydocstyle: 1.0.0, pycodestyle: 2.0.0, mccabe: 0.5.2, pyflakes: 1.2.3) CPython 3.5.1 on Darwin`
- Linter-flake8: `1.13.4`

**Code to reproduce**
```python
from sys import path as sys_path

# Nothing here, just need to room to move add lines
```

**Reported error**
`F401 — 'sys.path as sys_path' imported but unused at line 5 col 1 in test.py`

**Expected error**
`F401 — 'sys.path as sys_path' imported but unused at line 1 col 17 in test.py`
With the whole of `path as sys_path` underlined.

**Comment by Arcanemagus** [comment]
Just as an FYI, as part of the ES2017 re-write I'm working on I'm investigating this. I'm still not sure whether I want to keep the mass of code that is the "position fixing" and get it working properly, or just go back to the incorrect results from `flake8`...

**Comment by Arcanemagus** [comment]
Fixed in #255. Although the specs don't show it, it _does_ work with that syntax, it's just that the `hacking` module forces an older version of `flake8` that only marks the alias.
